.rubocop.yml

```diff
@@ -0,0 +1,7 @@
+AllCops:
+  Exclude:
+    - db/*.rb
+    - db/migrate/*.rb
```

Comment:
In that case, I would exclude all of `config` and include `config/seeds.rb`. That should work, but doublecheck.